### ISSUE TITLE
RegExp example

### ISSUE THREAD

**Issue by nicothin** [htmllint/htmllint#184]
in `.htmllint`:
```
{
  "class-style": "/[a-z]+/g"
}
```
in terminal:
```
[htmllint error in filePath.html ] TypeError: Cannot read property 'test' of undefined
```
please tell me where I go wrong?

**Comment by mickeyhead7** [comment]
The docs state you can set `An arbitrary Javascript RegExp object`, but I'm not sure how else you would format this as json in the .htmllintrc file if the format mentioned by @nicothin does not work?

**Comment by mickeyhead7** [comment]
@nicothin Update: this issue seems to have been worked on since the 0.6.0 tag release. Although another version has not yet been tagged (as of 3rd Feb 2017). If you are installing htmllint via NPM or using via [htmllint-cli](https://github.com/htmllint/htmllint-cli), then you will not be able to pass a regular expression in `.htmllintrc' until a new version is tagged (and htmllint-cli updated also).

**Comment by nicothin** [comment]
@mickeyhead7 i have `"htmllint-cli": "0.0.6"` via NPM

**Comment by mickeyhead7** [comment]
@nicothin Yep, 0.6.0 has a fault in `lib/knife/get_format_test.js` that can't accept a RegExp from the class-style` string in your `.htmllintrc` file. This has been removed/fixed/updated in the latest commit, but... that hasn't been tagged for release yet, so NPM and the CLI doesn't pull this in.

I'm having the same issue and haven't managed to test the latest version yet, so I am stuck as well :)

**Comment by mlochbaum** [comment]
I was aware of the JSON issue, but hadn't taken the time to figure out how RegExps should be specified by strings. Turns out it's pretty simple to just use Javascript's syntax (`/`, stuff, `/`, some combination of `gim`). I've excluded the `u` and `y` options because I'm pretty sure they aren't useful. `g` isn't either, but a lot of people use it for every regular expression, as this post shows.

I have some things I want to sort out before releasing 1.0.0, but I'll try to get it on npm this weekend.
